### PR TITLE
Adds integration testing infra

### PR DESCRIPTION
**Pull request by seunlanlege**

### CODE REVIEW

File: hyperspace/Makefile [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	docker run --platform linux/amd64 \
+	-d \
+	--rm \
+	-ti \
+	--name composable_centauri \
+	-u1000:1000 \
+	-p9944:9944 \
+	-p9188:9188 \
+	-p9988:9988 \
+	$(composable_centauri_image)
```
Comment (seunlanlege) [inline hyperspace/Makefile:18]: You probably want to run the `scripts/parachain-launch/docker-compose.yml` not just a single node running which won't produce any blocks

File: hyperspace/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+RUN curl -LO https://github.com/protocolbuffers/protobuf/releases/download/v3.12.4/protoc-3.12.4-linux-x86_64.zip && \
+    unzip protoc-3.12.4-linux-x86_64.zip -d .local && \
+    protoc --version && \
+    make build-release-hyperspace && \
+    ls -l ./target/release
+
+FROM debian:buster-slim
+
+COPY --from=backend-builder /usr/src/hyperspace-build/target/release/hyperspace .
+ENTRYPOINT ["./hyperspace"]
```
Comment (seunlanlege) [inline hyperspace/Dockerfile:19]: whats wrong with the old one?
https://github.com/ComposableFi/centauri/blob/seun%2Fintegration-test/scripts/hyperspace.Dockerfile

Comment (obsessed-cake) [reply hyperspace/Dockerfile:19]: ahh, I couldn't find it.
I'll fix it.

Comment (obsessed-cake) [reply hyperspace/Dockerfile]: fixed

**Review by Wizdave97 — APPROVED** [review]

**Comment by akc2267** [comment]
well done!!

**Pull request merged**